test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -81,6 +76,23 @@ def parse_apstat_rn(self):
         cmd_run = self.du.run_cmd(hostname, [apstat_cmd, options],
                                   as_script=True, wait_on_script=True,
                                   env=apstat_env)
+        return cmd_run
+
+    def parse_apstat_rn(self):
+        """
+        Run apstat command on cray/craysim and parse its output
+
+        :param options: options to pass to apstat command
```

Comment:
these two lines are not needed here

Reply:
updated the comment and removed param options which was a vestige from the existing code.